### DISCUSSION TITLE
评论区的自动展开可以关闭吗

### DISCUSSION THREAD

**Discussion by lostending** [the1812/Bilibili-Evolved#4548] [功能建议 / Ideas]
评论区的评论总是自动展开，在我重新安装过空白后仍然存在，这是内置的吗？可以关闭吗，非常多的回复层自动展开非常影响向下滑动观看其他评论，如果可以关闭就好了

**Comment by the1812** [comment]
没有这种功能

**Comment by Tinhone** [comment]
排查一下别的脚本？

**Comment by lostending** [comment]
呃呃，不好意思，确实是 ，我没想到不是这个脚本的问题，之前是某自动翻页的 锅，现在是东方永动页的问题，它们会把查看更多自动打开 问题 结束 ，感谢
